packages/smart-contracts/src/lib/artifacts/RequestHashStorage/index.ts

```diff
@@ -22,8 +22,12 @@ export const requestHashStorageArtifact = new ContractArtifact<RequestHashStorag
           creationBlockNumber: 4742809,
         },
         goerli: {
-          address: '0x132D0c7309Ca3286a644668469D3b09dFb81f757',
-          creationBlockNumber: 7145146,
+          address: '0x127eB88afD78EF627cAb2bD07ca5012Cd268a550',
+          creationBlockNumber: 8047209,
+        },
+        matic: {
+          address: '0x9aebb4b8abf7afc96dc00f707f766499c5ebedf1',
+          creationBlockNumber: 36486888,
```

Comment:
Please don't deploy `RequestHashStorage` and `RequestHashSubmitter` to Polygon (Matic)

We don't want to add new storage chains. The storage contracts already exist Gnosis Chain (xDai) and Ethereum.  Only the Gnosis Chain deployment is used and storage hashes on Ethereum are ignored.

Reply:
Will do